File: 프로그래머스/unrated/181854. 배열의 길이에 따라 다른 연산하기/배열의 길이에 따라 다른 연산하기.js
function solution(arr, n) {
    if (arr.length % 2 === 0) {
        return arr.map((el, i) => {
            return i % 2 === 1 ? el += n : el;
        });
    } else {
        return arr.map((el, i) => {
            return i % 2 === 0 ? el += n : el;
        });
    }
}

File: 프로그래머스/unrated/181854. 배열의 길이에 따라 다른 연산하기/배열의 길이에 따라 다른 연산하기.java
class Solution {
    public int[] solution(int[] arr, int n) {
        boolean isEven = arr.length % 2 == 0;
        int firstIndex = isEven ? 1 : 0;
        
        for (int i = firstIndex; i < arr.length; i += 2) {
            arr[i] += n;
        }
        
        return arr;
    }
}